File: server/src/main/java/org/cleanas2/bus/LoadCertificateMsg.java
package org.cleanas2.bus;

/**
 * @author Andrew Backer {@literal awbacker@gmail.com / andrew.backer@powere2e.com}
 */
public class LoadCertificateMsg extends MessageBase {
    public final String fileName;
    public final String alias;
    public final String password;

    public LoadCertificateMsg(String fileName, String alias) {
        this(fileName, alias, "");
    }

    public LoadCertificateMsg(String fileName, String alias, String password) {
        this.fileName = fileName;
        this.alias = alias;
        this.password = password;
    }
}

File: server/src/main/java/org/cleanas2/bus/LoadCertificateMsg.kt
package org.cleanas2.bus

/**
 * @author Andrew Backer awbacker@gmail.com / andrew.backer@powere2e.com
 */
class LoadCertificateMsg @JvmOverloads constructor(val fileName: String, val alias: String, val password: String = "") : MessageBase()

